src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs

```diff
@@ -382,6 +382,9 @@ public void CSharp_VerifyIDEDiagnosticSeveritiesAreConfigurable()
 # IDE0083
 dotnet_diagnostic.IDE0083.severity = %value%
 
+# IDE0085
+dotnet_diagnostic.IDE0085.severity = %value%
```

Comment:
adding a value below an existing value seems not good.  this should go after all the values we have.  we may have several in tihs bucket as well, so i would partition out a few vlaues here.

Reply:
Moved this to IDE0110